pgxmock.go

```diff
@@ -430,7 +430,7 @@ func (c *pgxmock) Query(ctx context.Context, sql string, args ...interface{}) (p
 		if err := c.queryMatcher.Match(queryExp.expectSQL, sql); err != nil {
 			return err
 		}
-		if err := queryExp.argsMatches(args); err != nil {
+		if err := queryExp.argsMatches(sql, args); err != nil {
```

Comment:
Why do we want `sql` parameter here? We already have `queryExp.expectSQL`

Reply:
I assumed that the expectSQL was the partial used to match the query (eg `^INSERT INTO users (.+)`) , not the original query which may or may not have named arguments. We need the original SQL to performed the named argument substitution operation.
Was my assumption incorrect?

Reply:
Oh, I see! This is the rewritten query. My bad. That returns us to the question should we store rewrites and try to allow expect them

Reply:
> That returns us to the question should we store rewrites and try to allow expect them.

It's a good question. I don't currently need that functionality, but I could see how someone else might desire that -- an example could be using `pg_query_go` to ensure the resulting sql is correct/conformant somehow (not sure how that could be done currently, but I could see how someone might be interested in that).